crates/electrum/Cargo.toml

```diff
@@ -9,12 +9,14 @@ description = "Fetch data from electrum in the form BDK accepts"
 license = "MIT OR Apache-2.0"
 readme = "README.md"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
 bdk_chain = { path = "../chain", version = "0.17.0" }
-electrum-client = { version = "0.20" }
-#rustls = { version = "=0.21.1", optional = true, features = ["dangerous_configuration"] }
+electrum-client = { version = "0.21", default-features = false }
 
 [dev-dependencies]
 bdk_testenv = { path = "../testenv", default-features = false }
+
+[features]
+default = ["use-rustls"]
+use-rustls = ["electrum-client/proxy", "electrum-client/use-rustls"]
```

Comment:
I agree with tnull's comment. I didn't notice it before, but I think we should also have it as the default as it's currently "mandatory/required" to properly work.

I don't think the way we have it on `rust-electrum-client` is the way to go, we should fix it there.

Reply:
Take a look at bitcoindevkit/rust-electrum-client#143. If that's the best way forward, we can update the bdk_electrum crate once that's merged (but maybe not in this PR? The bindings are stopped because of this so I'd like to get a fix in for the next beta release).